src/main/java/ch/wisv/areafiftylan/exception/TicketOptionNotFoundException.java

```diff
@@ -15,18 +15,10 @@
  *     along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package ch.wisv.areafiftylan.products.model;
+package ch.wisv.areafiftylan.exception;
 
-public enum TicketOptions {
-    PICKUPSERVICE(2.50F), CHMEMBER(-5.00F);
-
-    final float price;
-
-    TicketOptions(float price) {
-        this.price = price;
-    }
-
-    public float getPrice() {
-        return price;
+public class TicketOptionNotFoundException extends RuntimeException {
```

Comment:
Shouldn't this extend `IllegalArgumentException`? Also for the other exceptions down below